### DISCUSSION TITLE
Convert uphills from existing river rapids, log flumes and splash boats to use lifts.

### DISCUSSION THREAD

**Discussion by karst** [OpenRCT2/OpenRCT2#18294] [Feature requests]
I think it'd be a great addition to have the river rapids, log flumes and splash boats (help me if I missed something), similar to how the Alpine RC is currently implemented. That way we can set chainlift speed without cheats, add sounds for beltlifts aswell. With cheats we'd then be able to make uphill elements without a lift, similar to what you can find on rides like Chiapas at Phantasialand.

It is currently possible to recreate this by making two water of the same rides in opposing direction and then making one hidden, but an actual implementation would be nicer.

![image](https://user-images.githubusercontent.com/1516893/195158681-f63868ae-f733-42ad-874b-30978dfd3ad5.png)

**Comment by J-PIE-314** [comment]
PR https://github.com/OpenRCT2/OpenRCT2/pull/20258 is intended to add support for changing water ride lift speeds.

**Comment by J-PIE-314** [comment]
https://github.com/OpenRCT2/OpenRCT2/discussions/20180 is related to this request.
